### PR TITLE
Checks if the list of repositories is sorted in reverse alphabetical order

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #569

### CODE REVIEW

**Review by lubosmj — COMMENTED** [review]
@mdellweg, would it not be better to test the presence of repositories (i.e., "assert repositories_in_order.without(unknown_repositories) == 'aaa bbbb cccc'") instead of verifying whether repositories are in an equal order?

**Comment by mdellweg** [comment]
How about just looking at the output and see if it is alphabetically orderd?
`sort -c` should work.

**Comment by dkliban** [comment]
@mdellweg that's exactly what we are interested in knowing anyway. i'll update my PR to check if the list is sorted.

**Review by lubosmj — DISMISSED** [review]

File: tests/scripts/pulpcore/test_query_params.sh [hunk 1]
```diff
@@ -17,10 +17,8 @@ expect_succ pulp file repository create --name bbbbbbbbbb
 expect_succ pulp file repository create --name cccccccccc
 
 expect_succ pulp file repository list --ordering -name
-ORDERED_RESULTS=$(echo "$OUTPUT" | jq -r .[].name | tr "\n" " " | xargs)
-EXPECTED_RESULTS="cccccccccc bbbbbbbbbb aaaaaaaaaa"
-if [[ "$ORDERED_RESULTS" != "$EXPECTED_RESULTS" ]]; then
-  echo "Ordered results do not match: {$ORDERED_RESULTS} != {$EXPECTED_RESULTS}"
+if ! (echo "$OUTPUT" | jq -r .[].name | sort -r -C -); then
```
[the remark this answers is not in the document]
Comment (dkliban) [reply tests/scripts/pulpcore/test_query_params.sh:20]: Yes! I just added it.

**Pull request merged**